.github/workflows/dev-ecs.yml

```diff
@@ -24,9 +24,6 @@ jobs:
         uses: mbta/actions/commit-metadata@v1
       - uses: mbta/actions/build-push-ecr@v1
         id: build-push
-        env:
-          ERL_COOKIE: ${{ secrets.ERL_COOKIE }}
-          SECRET_KEY_BASE: ${{ secrets.SECRET_KEY_BASE }}
```

Comment:
Should the `docker-additional-args` value a few lines down from here be updated to not include `--build-arg ERL_COOKIE --build-arg SECRET_KEY_BASE` since these have been removed?

Reply:
oh yes, good call. I'll remove those!